.github/workflows/sonalCloud-build.yml

```diff
@@ -0,0 +1,20 @@
+name: SonarCloud Build 🚀
```

Comment:
.github/workflows/sonalCloud-build.yml
-> .github/workflows/SonarCloud-build.yml

Reply:
Is .github/workflows/sonarCloud-build.yml alright ?